src/cargo/util/toml/mod.rs

```diff
@@ -525,7 +525,8 @@ fn normalize_toml(
         normalized_toml.badges = original_toml.badges.clone();
     } else {
         if let Some(field) = original_toml.requires_package().next() {
```

Comment:
looking at https://github.com/rust-lang/cargo/blob/ce948f4616e3d4277e30c75c8bb01e094910df39/crates/cargo-util-schemas/src/manifest/mod.rs#L72, the suggestion doesn't apply to most fields.

Reply:
Ah 🤦‍♀️ I clearly have forgotten how little i remember around crate fields. Will close this for now as I'm not going to be able to look at it for a few weeks.